pinot-common/src/main/java/org/apache/pinot/common/metrics/ControllerGauge.java

```diff
@@ -72,6 +72,8 @@
   // Size of the last uploaded offline segment
   LAST_PUSHED_SEGMENT_SIZE("LastPushedSegmentSize", false),
 
+  // Estimated size of a table per replica. Counts both offline and realtime.
+  TABLE_ESTIMATED_SIZE_PER_REPLICA("bytes", false),
```

Comment:
I'm thinking keeping 2 metrics (estimated is a little bit misleading IMO):
`TABLE_TOTAL_SIZE_ON_SERVER` and `TABLE_SIZE_PER_REPLICA_ON_SERVER`

We may deprecate the `OFFLINE_TABLE_ESTIMATED_SIZE` and use `TABLE_TOTAL_SIZE_ON_SERVER` instead because it covers both offline and realtime tables

Reply:
I added the new metric `TABLE_TOTAL_SIZE_ON_SERVER` and the gauge value is set separate for offline and realtime table.